plugins/org.eclipse.osee.orcs.db/src/org/eclipse/osee/orcs/db/internal/sql/SelectiveArtifactSqlWriter.java

```diff
@@ -356,56 +356,18 @@ private void writeFieldsCommonTableExpression(String artWithAlias, String attsAl
       fieldAlias = startCommonTableExpression("fields");
       writeSelectAndHint();
       writeSelectFields(attsAlias, "*");
-      write(", 0 AS other_art_type_id FROM ");
+      write(" FROM ");
       write(attsAlias);
       write("\n UNION ALL\n ");
-      SelectiveArtifactSqlWriter relWriter = new SelectiveArtifactSqlWriter(this);
-      relWriter.relsAlias = relsAlias;
-      List<SqlHandler<?>> handlers = new ArrayList<SqlHandler<?>>();
-      handlers.add(new SqlHandler<Criteria>() {
-         private String artAlias;
-         @Override
-         public void addTables(AbstractSqlWriter writer) {
-            writer.addTable(relsAlias);
-            artAlias = writer.getMainTableAlias(OseeDb.ARTIFACT_TABLE);
-         }
-
-         @Override
-         public void addPredicates(AbstractSqlWriter writer) {
-            writer.writeEquals(relsAlias, "other_art_id", artAlias, "art_id");
-         }
-
-         @Override
-         public int getPriority() {
-            return SqlHandlerPriority.RELATED_TO_ART_IDS.ordinal();
-         }
-      });
-      relWriter.write(handlers);
-      write(relWriter.toSql());
+      writeSelectAndHint();
```

Comment:
Q: wouldn't this cause  if (relsAlias == null && rels2Alias == null) {
         writeSelectFields(artAlias, "art_id", artAlias, "art_type_id", txAlias, "app_id", txAlias, "transaction_id",
            txAlias, "mod_type", txAlias, "tx_current", txAlias, "gamma_id as art_gamma_id");
         if (OptionsUtil.getIncludeApplicabilityTokens(rootQueryData.getOptions())) {
            writeSelectFields(getMainTableAlias(OseeDb.OSEE_KEY_VALUE_TABLE), "value app_value");
         } to get hit multiple times? Is that intentional?

Reply:
hmmm... those don't get hit at all actually.